src/main/java/org/openrewrite/maven/AbstractRewriteBaseRunMojo.java

```diff
@@ -126,6 +137,76 @@ protected ResultsContainer listResults(ExecutionContext ctx) throws MojoExecutio
         }
     }
 
+    private static void configureRecipeOptions(Recipe recipe, Set<String> options) throws MojoExecutionException {
+        if (recipe instanceof CompositeRecipe && recipe.getRecipeList().size() == 1) {
+            // Unpack active recipe if it's a single recipe
+            recipe = recipe.getRecipeList().get(0);
+        }
```

Comment:
- No longer necessary after https://github.com/openrewrite/rewrite/pull/4349
```suggestion
```

Reply:
This will be the final touch; then we can finally see this resolved and update this section of the docs:
https://github.com/openrewrite/rewrite-docs/blob/9f8f51d17b28248624487eecd15d352fc123ca2e/reference/faq.md?plain=1#L136-L143